### PR TITLE
ISPN-12144 xsite doc updates

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-12144

### CODE REVIEW

**Comment by oraNod** [comment]
@rigazilla hey, is there a change needed in https://github.com/infinispan/infinispan-operator/blob/master/documentation/asciidoc/topics/ref_services.adoc ? is `$NAME-sites` still correct or is it now `xsite-service` only?

**Comment by rigazilla** [comment]
> @rigazilla hey, is there a change needed in https://github.com/infinispan/infinispan-operator/blob/master/documentation/asciidoc/topics/ref_services.adoc ? is `$NAME-sites` still correct or is it now `xsite-service` only?

Seems to be `<clustername>-site"

File: documentation/asciidoc/topics/con_xsite_replication.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+the {ocp} clusters in **NYC** and **LON**. {ispn_operator} then creates a cross-
+site replication service so {brandname} clusters can back up data across
+locations.
+
+{ispn_operator} selects one site master for each {brandname} cluster. The site
+master is the node that coordinates all backup requests through the cross-site
+replication service.
+
+If the current site master node becomes unavailable then {ispn_operator} finds
+a new coordinator node and makes it the site master. The cross-site replication
```
Comment (pruivo) [inline documentation/asciidoc/topics/con_xsite_replication.adoc:28]: I would remove the `coordinator` word and say the operator finds the newly elected SiteMaster.

File: documentation/asciidoc/topics/con_xsite_replication.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ifdef::productized[]
+image::xsite-rhdg.svg[]
+endif::productized[]
+
+{ispn_operator} uses a Kubernetes API to establish a secure connection between
+the {ocp} clusters in **NYC** and **LON**. {ispn_operator} then creates a cross-
+site replication service so {brandname} clusters can back up data across
+locations.
+
+{ispn_operator} selects one site master for each {brandname} cluster. The site
```
Comment (pruivo) [inline documentation/asciidoc/topics/con_xsite_replication.adoc:23]: not sure if "select" is the right verb. the SiteMaster is set by JGroups and the operator only finds it and tags it.

Comment (oraNod) [reply documentation/asciidoc/topics/con_xsite_replication.adoc]: thanks @pruivo I've updated.

**Comment by rigazilla** [comment]
merged thanks @oraNod

**Pull request merged**